2017/about-us.html

```diff
@@ -0,0 +1,191 @@
+    <meta property="og:description" content="2017年の東京Node学園祭は「Be More Global, Be More Interactive」というスローガンの元、普段日本では出会えないような国際的に活躍しているゲストを迎えて開催します。ただ聴講するだけではなく、双方向にコミュニケーションし合える場を提供します！">
+    <link rel="apple-touch-icon" sizes="180x180" href="img/icons/apple-touch-icon.png">
+    <link rel="icon" type="image/png" href="img/icons/favicon-32x32.png" sizes="32x32">
+    <link rel="icon" type="image/png" href="img/icons/favicon-16x16.png" sizes="16x16">
+    <link rel="manifest" href="manifest.json">
+    <link rel="mask-icon" href="img/icons/safari-pinned-tab.svg" color="#d75a43">
+    <link rel="shortcut icon" href="img/icons/favicon.ico">
+    <meta name="msapplication-config" content="browserconfig.xml">
+    <meta name="theme-color" content="#ffffff">
+    <link rel="stylesheet" href="main.css">
+  </head>
+  <body>
+
+<svg style="position: absolute; width: 0; height: 0; overflow: hidden;" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+  <defs>
+    <linearGradient id="mount-gradient" x1="0%" y1="0%" x2="0%" y2="100%">
+      <stop offset="0%"  stop-color="#D75A43" stop-opacity="0.7"></stop>
+      <stop offset="40%" stop-color="#F7DF1E" stop-opacity="0.9"></stop>
+      <stop offset="50%" stop-color="#EADA96"></stop>
+    </linearGradient>
+
+    <symbol id="mountain" id="corner" viewBox="0 0 2560 1478">
+      <path d="M2560 1478V739L1280 0 0 739v739"></path>
+    </symbol>
+
+    <symbol id="nav" viewBox="0 0 48 48">
+      <g fill-rule="evenodd"><path d="M0 0h48v8H0zM0 20h48v8H0zM0 40h48v8H0z"></path></g>
+    </symbol>
+
+    <symbol id="close" viewBox="0 0 48 48">
+      <path fill-rule="evenodd" d="M24 16.93L7.03-.043l-7.072 7.07L16.928 24-.04 40.97l7.07 7.072L24 31.072l16.97 16.97 7.072-7.07L31.072 24 48.04 7.03l-7.07-7.072L24 16.928z"></path>
+    </symbol>
+
+  </defs>
+</svg>
+
+<header id="header" class="header">
+  <div class="header-inner">
+    <a href="./" class="header-logo"><img src="img/logo-tnf-invert.svg" alt="東京Node学園祭2016"></a>
+    <nav class="navigation">
+      <ul class="navigation-list">
+        <li><a href="schedule.html" class="navigation-target">Schedule</a></li>
+        <li><a href="about-us.html" class="navigation-target">About us</a></li>
+      </ul>
+  </div>
+</header>
+
+<div class="main">
+  <section class="page schedule">
+    <h2 class="page-title team-title">About us</h2>
+    <div class="page-section">
+      <h3 class="page-subhead">Who we are?</h3>
+      <div class="page-sectionBody">
+        <ul class="team">
+          
+          <li>
+            <div class="team-member">
+              <p class="team-memberName"><a href="https://twitter.com/amagitakayosi" target="_blank">@amagitakayosi</a></p>
+            </div>
+          </li>
+          
+          <li>
+            <div class="team-member">
+              <p class="team-memberName"><a href="https://twitter.com/abouthiroppy" target="_blank">@abouthiroppy</a></p>
```

Comment:
my account is `twitter.com/about_hiroppy` 😂

Reply:
oops, sorry! will fix it.